{{cookiecutter.project_slug}}/README.md

```diff
@@ -63,11 +74,10 @@ mbed compile
 ```
 
 Program the target device with a Segger J-Link debug probe and
-[`sixtron_flash`](https://github.com/catie-aq/6tron_flash) tool:
+
+[`sixtron_flash`](https://github.com/catie-aq/6tron-flash) tool:
 ```shell
-sixtron_flash {{cookiecutter.__jlink_device}} BUILD/{{cookiecutter.__mbed_os_target}}/{{cookiecutter.toolchain}}/{{cookiecutter.project_slug}}.elf
+sixtron_flash
 ```
 
-Debug on the target device with the probe and Segger
-[Ozone](https://www.segger.com/products/development-tools/ozone-j-link-debugger)
-software.
+Debug on the target device with the probe and Segger [Ozone](https://www.segger.com/products/development-tools/ozone-j-link-debugger) software.
```

Comment:
I think if you think we should switch to longer lines instead of wrapped lines, you should create an issue, but not change it in this PR.

Reply:
Yes of course, if the issue is fixing typos, one PR is enough.
But in this case, one issue is enough too…